shibata added English-version file of the difference after February 10 (#110)
fujitake/vantiq-related: vantiq-external-services-integration/docs/en/vantiq-aws-iotcore.md

# Introduction
Describe how to collaborate with AWS IoT APIs directly from Vantiq.    
From Vantiq, the following collaborations are possible.    

- Receive data (telemetry) from Vantiq to devices    
  Can Pub/Sub to MQTT on AWS IoT Core by setting mutual authentication in MQTT Source.  

- Read/write property values (Registry)    
  Can only be operated from AWS Console | CLI | IoT API.    
  Can be done by calling AWS IoT API.    

> - Transmission of properties (Control Support)    

Reviewer: ```suggestion
- Transmit properties (Control Signal)    

```

  Use Device Shadow.  
  Device Shadow supports HTTPS | MQTT, both of which are possible from Vantiq.  

> In this document, describe how to do the following from Vantiq's Procedures.  

Reviewer: ```suggestion
This document describes how to do the following from Vantiq's Procedures.  

```

- Retrieve a list of "Things"  
- Retrieve details (Registry, Properties) of "Things"  

> - Retrieve properties (Control Support, Device Shadow) of "Things"  

Reviewer: ```suggestion
- Retrieve properties (Control Signal, Device Shadow) of "Things"  

```

- Update properties of "Things" (POST to Device Shadow)  

## Prerequisites
Vantiq Server v1.33 or higher is required

## Configuration of IoT Core
Prepare the Access key and the Secret key for your account to run AWS IoT Core REST API from Vantiq.  

> ## Creating "Things" for Vantiq assignment  

Reviewer: ```suggestion
## Creating "Things" to be assigned to Vantiq 

```

> Create "Things" with the AWS console for assignment to Vantiq in order to make a connection from Vantiq to AWS IoT Core's MQTT.    

Reviewer: ```suggestion
Create "Things" with the AWS console to be assigned to Vantiq in order to make a connection from Vantiq to AWS IoT Core's MQTT.    

```
